napari/layers/image/_tests/test_image.py

```diff
@@ -721,3 +722,41 @@ def test_2d_image_with_channels_and_affine_assignment_broadcasts(affine_size):
     image = Image(np.ones((1, 1, 1, 100, 100)))
     image.affine = np.eye(affine_size)
     np.testing.assert_array_equal(image.affine, np.eye(6))
+
+
+def test_image_state_update():
+    """Test that an image can be updated from the output of its
+    _get_state method()
+    """
+    image = Image(np.ones((32, 32, 32)))
+    state = image._get_state()
+    for k, v in state.items():
+        setattr(image, k, v)
+
+
+def test_instiantiate_with_plane_dict():
+    """Test that an image layer can be instantiated with plane parameters
+    in a dictionary.
+    """
+    plane_parameters = {
+        'position': (32, 32, 32),
+        'normal': (1, 1, 1),
+        'thickness': 22,
+    }
+    image = Image(np.ones((32, 32, 32)), plane=plane_parameters)
+    for k, v in plane_parameters.items():
+        if k == 'normal':
+            v = tuple(v / np.linalg.norm(v))
+        assert v == getattr(image.plane, k, v)
+
+
+def test_instiantiate_with_plane_manager():
+    """Test that an image layer can be instantiated with plane parameters
+    in a PlaneManager.
+    """
+    plane_parameters = PlaneManager(
```

Comment:
maybe change `plane_parameters` to `plane_manager` here, I was a bit confused on why this does not need normalization at first...